### ISSUE TITLE
Form is not 100% accessible for screen readers

### ISSUE THREAD

**Issue by ituk** [webvitalii/anti-spam#9]
Hey,
While checking my website accessibility on [WAVE](http://wave.webaim.org) I keep getting "missing form label" error for both text inputs (`antspm-q` and `antspm-e-email-url-website`). The label tags actually are there, so I guess adding a `for` attribute to each will solve the error.
I know the inputs are supposed to be hidden so screen reader should skip them, and I'm not an accessibility expert, but yet, WAVE returns that error over and over, so I'd appreciate if you check it out.

Thanks,
Itamar
